### PR TITLE
Add Plasma Leap spending conditions support

### LINKED ISSUE #84: Enable Execution of Spending Conditions

### PR DESCRIPTION
**Pull request by johannbarbie**
Add Plasma Leap spending conditions support

Resolves #84

### CODE REVIEW

File: src/tx/runCondition.test.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        prevout: new Outpoint(deposit.hash(), 0),
+        gasPrice: 100,
+        script: conditionScript,
+      }),
+    ],
+    [new Output(amount, `0x${receiver.toString('hex')}`, 1)]
+  );
+
+  // PRIV matches spenderAddr hardcoded in script
+  const PRIV =
```
Comment (sunify) [inline src/tx/runCondition.test.js]: Where PRIV would be stored?

**Comment by ryan-shea** [comment]
Hey @johannbarbie, Ryan from Gitcoin here. What is the status on this bounty? Wanted to check in and make sure everything is looking good. 

😄

**Comment by troggy** [comment]
still WIP?

**Comment by johannbarbie** [comment]
yes, @sunify proposed that i give the node a testrun with the new transaction :)

**Comment by CPSTL** [comment]
Hey @troggy @johannbarbie ! Gitcoin Ambassador here to check if the work has been reviewed yet?

File: tools/getProof.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const debug = require('debug')('getProof');
+const { helpers, Output, Outpoint, Tx } = require('leap-core');
+const { periodOfTheBlock } = require('./helpers/helpers');
+const { bufferToHex } = require('ethereumjs-util');
+const Web3 = require('web3');
+
+const web3 = new Web3('http://node1.testnet.leapdao.org:8645');
```
Comment (pinkiebell) [inline tools/getProof.js]: Switch to https load balancer? :)

Comment (johannbarbie) [reply tools/getProof.js]: i don't know that code & not needed, slipped in during rebase. removed.

**Pull request merged**